EnglishTranslation-master/SKILL.tsv

```diff
@@ -3083,7 +3083,7 @@ SKILL_20150323_003082	Death Sentence	죽음의 선고
 SKILL_20150323_003083	Prevented from becoming incapable of combat once.	전투불능상태를 한번 막음.				
 SKILL_20150323_003084	Throw left-hand weapon	왼손 장비 던지기				
 SKILL_20150323_003085	Resist Elements	레지스트 엘리멘츠				
-SKILL_20150323_003086	Critical rate is higher when attacking from behind.	뒤쪽에서 공격시 크리티컬 확률이 높음.				
+SKILL_20150323_003086	Critical Rate is higher when attacking from behind.	뒤쪽에서 공격시 크리티컬 확률이 높음.				
```

Comment:
Is this like the description when you hover on a buff?
What about: `Increased Critical Rate when attacking from behind` - Technically, it's correct since higher means there is an increase, right?